### ISSUE TITLE
Dashboard costs should not include costs of exported fuels

### ISSUE THREAD

**Issue by jorisberkhout** [quintel/etmodel#2204]
We now explicitly model the refinery products that are exported. As discussed with John, the costs of these fuels should not be included in the dashboard cost item.

**Comment by ChaelKruip** [comment]
I know that are are in a hurry but I'd like to know a bit more about the reasoning behind this decision and the original modeling. Can that be added at some point here for future reference?

**Comment by RobTerwel** [comment]
> I know that are are in a hurry but I'd like to know a bit more about the reasoning behind this decision and the original modelling.

I've submitted a PR in https://github.com/quintel/etsource/issues/1161 , which is in line with the original query. I however highly recommend having a look at 
- the purpose and effect of the original query
- *what costs associated with refineries should be included in the dashboard*
- whether point 1 lives up to point 2

If it doesn't, the query should be rewritten to include the right refinery costs.

**Comment by ChaelKruip** [comment]
Re-opening this for further evaluation but not under the Deploy milestone.

**Comment by ChaelKruip** [comment]
@mabijkerk can you pick this up?

**Comment by noracato** [comment]
Can this be closed now that the cost project (#3779 + quintel/etengine#1187) has been completed?

**Comment by mabijkerk** [comment]
Looking at the underlying query it seems as though this issue can indeed be closed @noracato. The main costs query for carriers, costs_carriers.gql looks like this:

```ruby
- query =
    SUM(
      Q(costs_carriers_biomass),
      Q(costs_carriers_oil_and_products),
      Q(costs_carriers_coal_and_products),
      Q(costs_carriers_uranium),
      Q(costs_carriers_heat),
      Q(costs_carriers_natural_gas),
      Q(costs_carriers_waste),
      Q(costs_carriers_electricity),
      Q(costs_carriers_hydrogen)
    )
```

Each query then sums the costs of imported carriers and **subtracts the costs of exported carriers**, for example for costs_carriers_uranium.gql:

```ruby
- query =
    SUM(
      PRODUCT(V(CARRIER(uranium_oxide), cost_per_mj), V(energy_extraction_uranium_oxide, demand)),
      PRODUCT(V(CARRIER(uranium_oxide), cost_per_mj), V(energy_import_uranium_oxide, demand))
    )-
    SUM(
      PRODUCT(V(CARRIER(uranium_oxide), cost_per_mj), V(energy_export_uranium_oxide, demand))
    )
```

Short testing with various carriers seems to confirm this behaviour.
